components/tidb_query_expr/src/impl_string.rs

```diff
@@ -1700,20 +1704,27 @@ mod tests {
     #[test]
     fn test_ord() {
         let cases = vec![
-            (Some("2"), Some(50i64)),
-            (Some("23"), Some(50i64)),
-            (Some("2.3"), Some(50i64)),
-            (Some(""), Some(0i64)),
-            (Some("你好"), Some(14990752i64)),
-            (Some("にほん"), Some(14909867i64)),
-            (Some("한국"), Some(15570332i64)),
-            (Some("👍"), Some(4036989325i64)),
-            (Some("א"), Some(55184i64)),
-            (None, Some(0)),
+            (Some("2"), Collation::Utf8Mb4Bin, Some(50i64)),
+            (Some("23"), Collation::Utf8Mb4Bin, Some(50i64)),
+            (Some("2.3"), Collation::Utf8Mb4Bin, Some(50i64)),
+            (Some(""), Collation::Utf8Mb4Bin, Some(0i64)),
+            (Some("你好"), Collation::Utf8Mb4Bin, Some(14990752i64)),
+            (Some("にほん"), Collation::Utf8Mb4Bin, Some(14909867i64)),
+            (Some("한국"), Collation::Utf8Mb4Bin, Some(15570332i64)),
+            (Some("👍"), Collation::Utf8Mb4Bin, Some(4036989325i64)),
+            (Some("א"), Collation::Utf8Mb4Bin, Some(55184i64)),
+            (Some("2.3"), Collation::Utf8Mb4GeneralCi, Some(50i64)),
```

Comment:
Please add both single byte and wide char cases under Latin1bin collation.

Reply:
OK, PTAL